src/test/run-make/rustdoc-test-builder/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# How to manually run this
+# $ ./x.py test src/test/run-make/rustdoc-test-builder
+
+all:
+	# See #81197 for why this uses `rustc` instead of `true`
+	# This needs to be a run-make test so rustc is available.
+	$(RUSTDOC) --test -Z unstable-options --test-builder $(RUSTC_ORIGINAL) doctest.rs
```

Comment:
Couldn't we just run it with `tee` or something?

Reply:
No, that doesn't accept all the options rustdoc passes: https://github.com/rust-lang/rust/pull/80924#discussion_r560334638